hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerRatisServer.java

```diff
@@ -310,9 +313,18 @@ private RaftClientRequest createRaftRequest(OMRequest omRequest) {
    */
   public OMResponse submitRequest(OMRequest omRequest,
       RaftClientRequest raftClientRequest) throws ServiceException {
-    RaftClientReply raftClientReply =
-        submitRequestToRatis(raftClientRequest);
-    return createOmResponse(omRequest, raftClientReply);
+    try {
+      CompletableFuture<OMResponse> reply = omRequestExecutor.submit(omRequest);
```

Comment:
isn't this handed off to another thread in submitToRatis ?

Reply:
yes and wait for ratis response